docs/embedding/full-app-embedding.md

```diff
@@ -4,161 +4,227 @@ redirect_from:
+**Full-app embedding** is the only type of embedding that integrates with your [permissions](../permissions/introduction.md) and [SSO](../people-and-groups/start.md#authentication) to give people the right level of access to [query](https://www.metabase.com/glossary/query_builder) and [drill-down](https://www.metabase.com/glossary/action_menu) into your data.
 
-## What you'll be doing
+## Prerequisites
 
-To get this going, you're going to need:
+1. Make sure you have a [license token](../paid-features/activating-the-enterprise-edition.md) for a [paid plan](https://store.metabase.com/checkout/login-details).
+2. Organize people into Metabase [groups](../people-and-groups/start.md).
+3. Set up [permissions](../permissions/introduction.md) for each group.
+4. Set up [SSO](../people-and-groups/start.md#authentication) to automatically apply permissions and show people the right data upon sign-in.
 
-- A [paid plan of Metabase](https://www.metabase.com/pricing) that includes full-app embedding.
-- A separate web application that you want to embed your dashboards and charts in.
+If you're dealing with a [multi-tenant](https://www.metabase.com/learn/customer-facing-analytics/multi-tenant-self-service-analytics) situation, check out our recommendations for [Configuring permissions for different customer schemas](https://www.metabase.com/learn/permissions/multi-tenant-permissions).
 
-## Enabling embedding in Metabase
+## Enabling full-app embedding in Metabase
 
-First, let's enable embedding in your Metabase instance. Go to the Admin Panel, and under Settings, go to the “Embedding in other applications” tab. From there, click “Enable.”
+1. Go to **Admin settings** > **Embedding**.
+2. Click **Enable**.
+3. Click **Full-app embedding**.
+4. Under **Authorized origins**, add the URL of the website or web app where you want to embed Metabase.
 
-Once you do, you'll see a set of options:
+## Setting up embedding on your website
 
-- **Embedding secret key:** You can ignore this setting, which is only for standalone chart or dashboard embeds.
+1. Create an iframe with a `src` attribute set to:
+   - the [URL](#pointing-an-iframe-to-a-metabase-url) of the Metabase page you want to embed, or
+   - an [authentication endpoint](#pointing-an-iframe-to-an-authentication-endpoint) that redirects to your Metabase URL.
+2. Optional: Depending on the way your web app is set up, set [environment variables](../configuring-metabase/environment-variables.md) to:
```

Comment:
@flamber @paoliniluis @Tony-metabase 

Could you guys check the correctness of the sections under 2 (env vars) and 3 (JavaScript stuff) by the end of the week?

I did lots of reading, but still have the least confidence in translating those use cases correctly.

If there's no comments by the end of the week, I'll assume that it's good to go :) Thanks!